nuki/__init__.py

```diff
@@ -156,13 +158,15 @@ def parse_item(self, item):
 
             if self.has_iattr(item.conf, 'nuki_trigger'):
                 nuki_trigger = self.get_iattr_value(item.conf, "nuki_trigger")
-                if nuki_trigger.lower() not in ['state', 'action', 'battery']:
+                if nuki_trigger.lower() not in ['state', 'doorstate', 'action', 'battery']:
                     self.logger.warning("Plugin '{pluginname}': Item {item} defines an invalid Nuki trigger {trigger}! "
-                                        "It has to be 'state' or 'action'.".format(pluginname=self.get_shortname(),
-                                                                                   item=item, trigger=nuki_trigger))
+                                        "It has to be 'state', 'doorstate' or 'action'.".format(pluginname=self.get_shortname(),
```

Comment:
the indentation looks a little strange. Did you check it with a tool?

Reply:
No, I did not use a tool. Are you referring to the next line? I added blanks to align the part "item=item, ..." with the function call in the previous line.